src/main/java/de/retest/surili/commons/ga/TestLengthFitness.java

```diff
@@ -0,0 +1,21 @@
+package de.retest.surili.commons.ga;
+
+import de.retest.surili.commons.core.TestSuite;
+import lombok.NonNull;
+import lombok.Value;
+
+/**
+ * Length of all test cases of one test suite. Smaller tests produce a smaller fitness value which is better.
+ */
+@Value
+public class TestLengthFitness implements Fitness {
+
+	@NonNull
+	TestSuite testSuite;
+
+	@Override
+	public double getFitness() {
+		return testSuite.getTestCases().stream().map( testCase -> testCase.getActions().size() ).reduce( 0,
```

Comment:
Bit more idiomatic:

```java
return testSuite.getTestCases().stream() //
		.mapToInt( testCase -> testCase.getActions().size() ) //
		.sum();
```

We could also extend the `TestCase` and `TestSuite` interfaces:

```java
// TestCase
default int getSize() {
	return getActions().size();
}

// TestSuite
default int getSize() {
	return getTestCases().stream() //
			.mapToInt( TestCase::getSize ) //
			.sum();
}
```

Then the fitness calculation would be:

```java
return testSuite.getSize();
```